os/board/artik05x/src/artik05x_tash.c

```diff
@@ -171,36 +171,18 @@ int board_app_initialize(void)
 	int ret;
 #ifdef CONFIG_FLASH_PARTITION
 	struct mtd_dev_s *mtd;
-#if defined(CONFIG_ARTIK05X_AUTOMOUNT) && defined(CONFIG_RAMMTD) && defined(CONFIG_FS_SMARTFS)
+	partition_info_t partinfo;
+#if defined(CONFIG_AUTOMOUNT) && defined(CONFIG_RAMMTD) && defined(CONFIG_FS_SMARTFS)
 	int bufsize;
 	static uint8_t *rambuf;
 #endif /* CONFIG_RAMMTD */
 
 	mtd = (FAR struct mtd_dev_s *)mtd_initialize();
 	/* Configure mtd partitions */
-	configure_mtd_partitions(mtd, &g_flash_part_data);
+	configure_mtd_partitions(mtd, &g_flash_part_data, &partinfo);
 
-#ifdef CONFIG_ARTIK05X_AUTOMOUNT
-#ifdef CONFIG_ARTIK05X_AUTOMOUNT_USERFS
-	/* Initialize and mount user partition (if we have) */
-#ifdef CONFIG_SMARTFS_MULTI_ROOT_DIRS
```

Comment:
This is disappeared in new function. Could you check this?

Reply:
Done. Thanks.